docs/hazmat/primitives/asymmetric/ec.rst

```diff
@@ -230,6 +230,13 @@ All named curves are providers of :class:`EllipticCurve`.
     SECG curve ``sect283k1``. Also called NIST K-283.
 
 
+.. class:: SECT239K1
+
+    .. versionadded:: 1.2
+
+    SECG curve ``sect239k1``.
```

Comment:
I assume you have some need for these curves? Do we have any source for test vectors for this and `sect163r1`?